docs/user-guide/openai-api/translation-agent.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 5
+---
+
+# Translation Agent on LlamaEdge
+
+This LLM Translation Agent originally built by [Prof. Andrew Ng](https://www.linkedin.com/posts/andrewyng_github-andrewyngtranslation-agent-activity-7206347897938866176-5tDJ/) is designed to facilitate accurate and efficient translation across multiple languages. It employs open source LLMs (Large Language Models) to provide high-quality translations. You can use your own fine-tuned models or any LLMs on Hugging Face like Meta's Llama 3. This documentation shows how the Transgenic Agent utilizes the Gemma-2-9B model for translation.
+
+
+>For commands on starting and running this agent, refer to [GitHub - Second State/translation-agent](https://github.com/second-state/translation-agent/blob/use_llamaedge/step-by-step-use-LocalAI.md).
+
+
+## Run Gemma-2-9B on your own device
+
+See detailed instructions to [Run Gemma-2-9B on your own device.](https://www.secondstate.io/articles/gemma-2-9b/)
+
+Download the [Gemma-2-9B-it model GGUF file](https://huggingface.co/second-state/gemma-2-9b-it-GGUF). Since the size of the model is 6.40G, it could take a while to download.
+
+```
+curl -LO https://huggingface.co/second-state/gemma-2-9b-it-GGUF/resolve/main/gemma-2-9b-it-Q5_K_M.gguf
+```
+
+Start an API server for the model.
+
+```
+wasmedge --dir .:. --nn-preload default:GGML:AUTO:gemma-2-9b-it-Q5_K_M.gguf \
+  llama-api-server.wasm \
+  --prompt-template gemma-instruct \
+  --ctx-size 4096 \
+  --model-name gemma-2-9b
+```
+
+## Run the Translation Agent to run on top of Gemma-2-9B
+
+To get started, clone the Translation Agent.
+
+```
+git clone https://github.com/second-state/translation-agent.git
+    
+cd translation-agent
+git checkout use_llamaedge
+```
+
+Here, we run Gemma-2-9B locally and our Translation Agent on top of them respectively to showcase their translation quality. We test a simple translation task to see the results so as to compare their translation capabilities. You will need to install [WasmEdge](https://github.com/WasmEdge/WasmEdge) and the [LlamaEdge API server](https://github.com/LlamaEdge/LlamaEdge) to run those models across major GPU and CPU platforms.
```

Comment:
You have the full command line to run a open-source model in the previous paragraph. why mention downloading WasmEdge and llama-api-server.wasm here?